src/app/globals.css

```diff
@@ -9,3 +9,14 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+h1,
```

Comment:
This looks very specific to abouts page.

Maybe let's create a css/module for about and put it in there.

Eventually we need to migrate css into scss but we can worry about that later.